packages/database/src/migrations/20210124055714-ConvertDhisSurveysIntoTupaia-modifies-data.js

```diff
@@ -80,6 +87,15 @@ exports.up = async function (db) {
     await switchServiceType(db, code, 'dataGroup', serviceType);
   }
 
+  // Have to convert these reports to use 'Yes' because the data elements are Condition questions
+  dashboardReportsChangeForValueOfInterest.forEach(async id => {
+    await db.runSql(`
+      update "dashboardReport" dr 
+      set "dataBuilderConfig" = regexp_replace(dr."dataBuilderConfig"::text, '\\"valueOfInterest\\"\\: 1','"valueOfInterest": "Yes"','g')::jsonb 
+      where id = '${id}'
+  `);
+  });
```

Comment:
`RHS6UNFPA1354` and `RHS6UNFPA1355` have been converted from `Binary` to `Condition` questions last week (https://github.com/beyondessential/tupaia-backlog/issues/1359). Those 2 questions are now hidden and generate Yes/No automatically. 

Since they're no longer `Binary`, they have not been sanitized to `1/0`, causing this bug here (https://github.com/beyondessential/tupaia-backlog/issues/1322#issuecomment-778969387).

I feel that this is probably a special case and won't happen so often so it's fine to just modify the reports to use `Yes/No` instead of trying to find a way to also uniform them to be `1/0`

Reply:
Darn!